spec/javascripts/govuk_publishing_components/analytics-ga4/gtm-page-views.spec.js

```diff
@@ -247,4 +247,34 @@ describe('Google Tag Manager page view tracking', function () {
     GOVUK.Gtm.sendPageView()
     expect(window.dataLayer[0]).toEqual(expected)
   })
+
+  it('removes email pii from the title, location and referrer', function () {
+    document.title = 'example@gov.uk'
+    expected.page.title = '[email]'
+
+    spyOnProperty(document, 'referrer', 'get').and.returnValue('https://gov.uk/example@gov.uk')
+    expected.page.referrer = 'https://gov.uk/[email]'
+
+    // We can't spy on location, so instead we use an anchor link to change the URL temporarily
+
+    // Reset the URL so we can build the expected link string
+    const linkForURLMock = document.createElement('a')
```

Comment:
Historically we've avoided using `const` as it isn't supported in IE10 or below. Obviously the tests don't run in users browsers, so it's safer here, but ideally the tests should pass in older browsers as well so I think maybe sticking with `var` is safer.